### PR TITLE
Add crossdock tests to travis

### PR DESCRIPTION
**Pull request by black-adder**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -38,6 +42,8 @@ before_install:
   - sudo apt-get update -qq
   - sudo apt-get install -y -qq yarn
   - nvm install 6
+  - chmod +x ./travis/install-crossdock-deps.sh # TODO exec mode set on the file in git doesn't work as intended
```
Comment (yurishkuro) [inline .travis.yml]: you could just do `bash ./travis/install-crossdock-deps.sh`

**Comment by yurishkuro** [comment]
how fast are these tests?

**Comment by black-adder** [comment]
I'll let you know as soon as a travis box becomes available

**Comment by black-adder** [comment]
crossdock takes roughly 5 minutes which the same amount of time as the tests. If we're lucky and everything runs in parallel, 5 minutes for everything. If super unlucky, 15 minutes.

**Pull request merged**